personal_profile/tests.py

```diff
@@ -104,3 +105,29 @@ def test_detail_view_template(self, profile_1, user_1, client):
         client.force_login(test_user)
         response = client.get(PROFILE_DETAIL_URL + str(profile_1.id) + '/')
         assertTemplateUsed(response, 'personal_profile/personalprofile_detail.html')
+
+
+@pytest.mark.django_db
+class TestProfileUpdateView:
+    def test_update_view_profile(self, profile_1, user_1, client):
+        # Testing to see if the changes made for company, birth_date and about fields
+        # will result with response code of 302
+        test_profile = PersonalProfile.objects.filter(user=user_1).first()
+        test_user = User.objects.filter(username='user_1').first()
+        client.force_login(test_user)
+        update_profile_path = PROFILE_DETAIL_URL + str(test_profile.id) + UPDATE_PROFILE_PATH
+        new_fields = {
+            'company': 'Update Test Company',
+            'birth_date': datetime.date(1992, 12, 10),
+            'about': 'Update Test About',
+            'profile_pic': test_profile.profile_pic,
+            'resume': test_profile.resume}
+        response = client.post(update_profile_path, new_fields)
+        assert response.status_code == 302
```

Comment:
Please check the redirect uri of the response with another request(response.uri) 
Assert the 200 returned and templates